default.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
   <manifest>
-  <remote fetch="https://github.com/nxp-auto-linux" name="alb"/>
+  <remote fetch="https://github.com/consider-it/meta-alb" name="alb"/>
```

Comment:
We might not even need this change as all forks share all commits at GitHub. But don't know what the repo tool does exactly.

E.g. you can clone https://github.com/nxp-auto-linux/meta-alb, do `git fetch 981d365b4385d5b72502a9035cb26282a926deed` (revision listed below) and `git reset --hard` to it.

Reply:
i just made this change to be coherant with what we did (witout taking into account that the work will be sent to bitbucket server)

So yes as you got point, but once the meta-alb commits are merged by NXP in their server.